public/app/features/alerting/unified/state/AlertingQueryRunner.ts

```diff
@@ -44,24 +45,33 @@ export class AlertingQueryRunner {
   }
 
   async run(queries: AlertQuery[]) {
-    if (queries.length === 0) {
-      const empty = initialState(queries, LoadingState.Done);
-      return this.subject.next(empty);
-    }
+    const empty = initialState(queries, LoadingState.Done);
+    const queriesToExclude: string[] = [];
 
     // do not execute if one more of the queries are not runnable,
-    // for example not completely configured
+    // for example not completely
```

Comment:
`configured` word missing in this comment?

Reply:
Good catch! Not sure how that happened but will definitely revert this